index.html

```diff
@@ -1,14 +1,45 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Vanilla Todo</title>
-    <link rel="stylesheet" href="style.css" />
-  </head>
-
-  <body>
-    <div class="container"></div>
-  </body>
-  <script src="script.js"></script>
-</html>
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>To-do list</title>
+    <link rel ="stylesheet" href ="style.css"/>
+    <script src="https://kit.fontawesome.com/f9b359f751.js" crossorigin="anonymous"></script>
+
+
+
+</head>
+<body>
+    <div class ="whiteBox">
+
+
+        <div class = "inputBox">
+             <div class ="toDoList title-font">❗투두리스트❗</div>
+             <div class ="toDoInput">
+                 <input id ="toDo" placeholder="할 일을 입력하세요"></input>
+                 <i id ="toDoButton" class="fa-solid fa-plus"></i>
+            </div>
+        </div>
+
+        <div class = "toDoBox">
+            <div id = todo-list>
+                  <div class ="title-font">✍️To Do<span id = "toDoInfo" >(0)</span></div>
+                  <div id ="finishtoDo" class ="hidden">과제끝!!!🤟</div>
```

Comment:
```suggestion
                  <div id ="finishtoDo" hidden>과제끝!!!🤟</div>
```
이런 식으로 attribute로 element를 숨길 수도 있답니다.

Reply:
조언 감사합니다 그대로 반영하겠습니다!!